lib/dialects/mssql/blocks.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var _ = require('underscore');
+
+module.exports = function(dialect) {
+	dialect.blocks.set('limit', function(params) {
+		return (!isNaN(params.offset)) ? '' : 'top(' + dialect.builder._pushValue(params.limit) + ')';
+	});
+
+	dialect.blocks.set('offset', function(params) {
+		var pre = (!params.sort) ? 'order by 1 ' : '';
+		if (params.limit) {
+			var str = pre + 'offset ' + dialect.builder._pushValue(params.offset);
+			str += ' rows fetch next ' + dialect.builder._pushValue(params.limit) + ' rows only';
+			return str;
+		}else {
+			return pre + 'OFFSET ' + dialect.builder._pushValue(params.offset) + ' rows';
```

Comment:
all words should be in lower case, you use `offset` word above but here `OFFSET`